packages/eds-core-react/src/components/Popover/Popover.stories.tsx

```diff
@@ -320,6 +321,79 @@ WithCloseButton.decorators = [
   },
 ]
 
+export const NotDismissable: Story<PopoverProps> = () => {
+  const Wrapper = styled.div`
+    display: grid;
+    grid-gap: 6px;
+    :last-child {
+      padding-bottom: 6px;
+    }
+  `
+
+  const counties = [
+    'Oslo',
+    'Rogaland',
+    'Møre og Romsdal',
+    'Nordland',
+    'Viken',
+    'Innlandet',
+    'Vestfold og Telemark',
+    'Agder',
+    'Vestland',
+    'Trøndelag',
+    'Troms og Finnmark',
+  ]
+
+  const [isOpen, setIsOpen] = useState<boolean>(false)
+  const anchorRef = useRef<HTMLButtonElement>(null)
+
+  const openPopover = () => setIsOpen(true)
+  const closePopover = () => setIsOpen(false)
+
+  return (
+    <>
+      <Button
+        id="click-popover-anchor"
+        aria-controls="click-popover"
+        ref={anchorRef}
+        onClick={openPopover}
+      >
+        Click to activate
+      </Button>
+
+      <Popover
+        id="click-popover"
+        aria-expanded={isOpen}
+        anchorEl={anchorRef.current}
+        open={isOpen}
+      >
+        <Popover.Header>
+          <Popover.Title>With Autocomplete</Popover.Title>
+        </Popover.Header>
+        <Popover.Content>
+          <Wrapper>
+            <Autocomplete label="Select a counties" options={counties} />
+            <Autocomplete label="Select a counties" options={counties} />
```

Comment:
Typo: "Select a county"
Do we need two autocompletes in here?

Reply:
Ditto, suggest just having the one `Autocomplete` and removing the `Wrapper` to avoid confusion.